animations-baker.js

```diff
@@ -244,6 +246,7 @@ const {CharsetEncoder} = require('three/examples/js/libs/mmdparser.js');
 
       // scale position tracks by height
       for (const track of animation.tracks) {
+        if (track.name.indexOf('mixamorig') !== 0) track.name = 'mixamorig' + track.name;
```

Comment:
I downloaded `Standing Aim Walk Forwad/Left/Right/Back` from https://www.mixamo.com/ directly, they lack the `mixamorig` prefix.

I don't know how to batch rename the bone names in Blender, so modified them in `animation-baker.js`.
Should I request artists to rename them in Softwares?

Reply:
https://github.com/webaverse/app/pull/2903/commits/12c0ce5edc637feff3d3cd0e4f110a2c7b73c2f0

Used new bow walk animations by textametron, which has correct track name,
prevented modify the name in `animations-baker.js`. 

But separated `_setAnimationDuration()` from `_normalizeAnimationDurations()` and used it to modify the baseAnimaton's duration.
I think this gets codes more flexibility, is it ok?
Or have to modify the `.fbx` file or modify in `animations-baker.js`?

Reply:
Used already correct duration bow walk animations by textametron, instead of `_setAnimationDuration()`. https://github.com/webaverse/app/pull/2903/commits/589f3802f11dde7c8932d9577d7ea2591f70edcd